packages/radix-vue/src/Listbox/ListboxVirtualizer.vue

```diff
@@ -21,12 +21,15 @@ import { getNextMatch } from '@/shared/useTypeahead'
 import { useParentElement } from '@vueuse/core'
 import { useCollection } from '@/Collection'
 import type { AcceptableValue } from '@/shared/types'
+import type { Virtualizer, VirtualItem } from '@tanstack/vue-virtual'
 
 const props = defineProps<ListboxVirtualizerProps<T>>()
 
 defineSlots<{
   default: (props: {
-    option: T
+    option: T;
+    virtualizer: Virtualizer<Element | Window, Element>;
+    virtualItem: VirtualItem
```

Comment:
seems like it's failing due to `VirtualItem` requires 1 type argument, which is `Element`. Let's add this argument ya

https://github.com/radix-vue/radix-vue/actions/runs/10722138349/job/29756726077#step:8:32

Reply:
fixed (with lint in the imports too...)